catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -137,20 +241,47 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            json-smart,
-                            catalog-core-api-impl,
+                            asm,
                             catalog-core-actions,
-                            platform-util,
+                            catalog-core-api-impl,
+                            common-system,
                             commons-lang3,
-                            asm
+                            json-smart,
+                            platform-util
                         </Embed-Dependency>
                         <Import-Package>
+                            !org.codice.ddf.platform.util,
                             javax.ws.rs.*,
                             *
                         </Import-Package>
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <!-- Need to use an older version of failsafe for versionAsInProject() to work -->
+                <version>2.17</version>
```

Comment:
✏️ Since we need this workaround in a few places, can we extract the variable as something like `failsafe.versionAsInProjectWorks.version`?

Reply:
Added `<failsafe.pax-exam-compatible.version>` property to root pom file and updated other pom files accordingly.